src/lib.rs

```diff
@@ -1620,6 +1621,32 @@ fn weight_callable(
     }
 }
 
+#[pyfunction(normalized = "true", endpoints = "false")]
+#[text_signature = "(graph, /, normalized=True, endpoints=False)"]
+pub fn graph_betweenness_centrality(
+    py: Python,
+    graph: &graph::PyGraph,
+    normalized: bool,
+    endpoints: bool,
+) -> PyResult<PyObject> {
+    let betweenness =
+        centrality::betweenness_centrality(&graph, endpoints, normalized);
+    Ok(PyList::new(py, betweenness).into())
```

Comment:
One thing here is you can return `Ok(betweeness)` directly, the `pyfunction` macro will understand how to convert that into a python list (because there is an implementation of the `ToPythonObject` trait for Vec<T> where T also implements it). But it's probably worth benchmarking first because I've seen situations where constructing a python type explicitly in the rust code is faster than relying on the macro to do it for us (and vice versa).